packages/xeus-core/src/worker.base.ts

```diff
@@ -272,6 +281,11 @@ export abstract class XeusRemoteKernelBase {
         case 'list':
           await this.listInstalledPackages();
           break;
+        case 'remove':
+        case 'uninstall':
+          const { specs, env } = command.data as IUninstallationCommandOptions;
+          await this.uninstall(specs, env, command.type);
```

Comment:
No need for this third parameter, it's always "uninstall"?

```suggestion
          await this.uninstall(specs, env);
```

Reply:
@martinRenou thank you for taking time to check this PR:) No, type is important and it is different for conda and pip. For example `%conda remove numpy` or `%pip uninstall ipycanvas`. And using a command type helps separate deleting logic for pip and conda packages which has a few differences 
![image](https://github.com/user-attachments/assets/ab47a444-4bd9-4a42-a0ec-650dee239fb9)